{
  "latest_version": [
    "2.1.3"
  ],
  "meta": {
    "description": "KeyError.com Django client",
    "homepage": "https://keyerror.com/docs/django"
  },
  "versions": {
    "2": {
      "sha256": "c1f507d69b72545f50af83ef686d38bf27d0cef258842b4f8d24d0de6e6d9b2d",
      "url": "https://files.pythonhosted.org/packages/fd/a6/205a5a8c8a83a8e1079011aa2af94cbc11d4383ae6536d58675bf48f4057/django-keyerror-2.tar.gz"
    },
    "2.0.1": {
      "sha256": "8b4bda8489302b4c5f0d5009ed1d7b1299865b22910c9353b5dc47e74628fc69",
      "url": "https://files.pythonhosted.org/packages/0f/37/8ea0e249b6d3c9095052039d8fe44e4eafabd6681591e2f92ee4e7ef9699/django-keyerror-2.0.1.tar.gz"
    },
    "2.1.0": {
      "sha256": "e32b161053a158936b0f80801901a51dbab768c977826d7a3d937f9563524290",
      "url": "https://files.pythonhosted.org/packages/52/b2/22c69b49baed811fd03e5fea10cb3c2935854da60ae9d8dbd54aafabd6c1/django-keyerror-2.1.0.tar.gz"
    },
    "2.1.1": {
      "sha256": "dc9d1d79da4260dd45e7265437373cd96f50d583ac262d158ff6753d21f2ec37",
      "url": "https://files.pythonhosted.org/packages/03/22/913a19ae835e83ef06cf71aaca9b8d81b0394ba56b433be59f091ff865f0/django-keyerror-2.1.1.tar.gz"
    },
    "2.1.2": {
      "sha256": "627c5e84587e24038f3e1756f8f285abb86e1ce4dc979f253ad44cee792d15c5",
      "url": "https://files.pythonhosted.org/packages/94/2f/78f68a3abcc89f94d688c7c08f91a4d75e483c6609707305f2c992296197/django-keyerror-2.1.2.tar.gz"
    },
    "2.1.3": {
      "sha256": "97533bf82e12e78fd777d90e9d4ddedafd769e2e7549fc4b9de7258fddd6fea3",
      "url": "https://files.pythonhosted.org/packages/cb/3c/7fba828064bfe6cbbc61d6f4a03790a25c5d4faf7b89b8bca908eb3c012a/django-keyerror-2.1.3.tar.gz"
    }
  }
}